app/assets/stylesheets/geoblacklight/modules/_blacklight_overrides.scss

```diff
@@ -45,3 +45,7 @@ $logo-image: image_url('blacklight/logo.svg') !default;
   padding-top:0.5rem;
 }
 // End - Addresses GBL issue #639
+
+.navbar-search.bg-light {
+  background-color: inherit!important;
+}
```

Comment:
why the `!important`? does something else end up overriding this later?

Reply:
Yeah the default in blacklight is a whitish color.